### DISCUSSION TITLE
I desperatelty need your help. using revancedphotos. can i safely use it as secondary gmail account on same device (using primary account) ,without any chance of ban?

### DISCUSSION THREAD

**Discussion by cuirousprogrammer** [ReVanced/revanced-discussions#1082] [Question]
I have one android phone. I will install revancedPhotos with my secondary gmail (mircog) and will also use primary gmail (seperatly) but on the same android device. if my revnacedPhoto (Secondary) gets banned by google, will google can also ban my primary account (same device,same ip address). because if that happens, iam doomed.

**Comment by LisoUseInAIKyrios** [comment]
https://github.com/orgs/ReVanced/discussions/1073

If you're really concerned then don't use the patch.  Paying for a large storage plan is not that expensive.
